keyboards/keycapsss/kimiko/kimiko.c

```diff
@@ -75,4 +77,237 @@ led_config_t g_led_config = { {
     4, 4, 1, 1, 1, 1
 } };
 
-#endif // RGB_MATRIX_ENABLE
\ No newline at end of file
+#endif // RGB_MATRIX_ENABLE
+#endif // KEYBOARD_keycapsss_kimiko_rev1
+
+#ifdef KEYBOARD_keycapsss_kimiko_rev2
+enum layers {
+   _QWERTY,
+   _LOWER,
+   _RAISE,
+   _ADJUST
+};
+
```

Comment:
```suggestion
```

Reply:
This changes break the Rev2 default keymap.
I will revert it.